pom.xml

```diff
@@ -51,6 +51,10 @@
             <artifactId>spring-boot-starter-actuator</artifactId>
             <version>${springBootVersion}</version>
             <exclusions>
+                <exclusion>
```

Comment:
No need to add these changes as spring 2.6.3 implements logback version 1.2.10 .Please close the pull request